docs/api/create_docker_container.md

```diff
@@ -0,0 +1,78 @@
+| `WithWorkingDirectory`                  | Specifies or overrides the `WORKDIR` for the instruction sets.                                                  |
+| `WithEntrypoint`                        | Specifies or overrides the `ENTRYPOINT` that runs the executable.                                               |
+| `WithCommand`                           | Specifies or overrides the `COMMAND` instruction provided in the Dockerfile.                                    |
+| `WithEnvironment`                       | Sets an environment variable in the container e.g. `-e, --env "MAGIC_NUMBER=42"`.                               |
+| `WithExposedPort`                       | Exposes a port inside the container e.g. `--expose=80`.                                                         |
+| `WithPortBinding`                       | Publishes a container port to the host e.g. `-p, --publish 80:80`.                                              |
+| `WithBindMount`                         | Binds a path of a file or directory into the container e.g. `-v, --volume .:/tmp`.                              |
+| `WithVolumeMount`                       | Mounts a managed volume into the container e.g. `--mount type=volume,source=my-vol,destination=/tmp`.           |
+| `WithTmpfsMount`                        | Mounts a temporary volume into the container e.g. `--mount type=tmpfs,destination=/tmp`.                        |
+| `WithNetwork`                           | Assigns a network to the container e.g. `--network="bridge"`.                                                   |
+| `WithNetworkAliases`                    | Assigns a network-scoped aliases to the container e.g. `--network-alias alias`.                                 |
+| `WithPrivileged`                        | Sets the `--privileged` flag.                                                                                   |
+| `WithCreateContainerParametersModifier` | Allows low level modifications of the Docker container create parameter.                                        |
+| `WithOutputConsumer`                    | Redirects `stdout` and `stderr` to capture the container output.                                                |
+| `WithWaitStrategy`                      | Sets the wait strategy to complete the container start and indicates when it is ready.                          |
+| `WithStartupCallback`                   | Sets the startup callback to invoke after the container start.                                                  |
+
+!!!tip
+
+    Testcontainers for .NET detects your Docker host configuration. You do **not** have to set the Docker daemon socket.
+
+
+!!!tip
+
+    Testcontainers for .NET detects private Docker registry configurations and applies the credentials automatically to authenticate against registries.
```

Comment:
Maybe better to move it above the table.

Reply:
Straight under the headline?